src/bookshelf/custom.mk

```diff
@@ -36,3 +40,13 @@ clean_ct:
 
 ## Pull in devvm.mk for relxy goodness
 include devvm.mk
+
+version_clean:
+	@rm -f VERSION
+
+VERSION: version_clean
+ifeq ($(REL_VERSION),)
+	@echo -n "$$(git log --oneline --decorate | grep -F "tag: " --color=never | head -n 1 | sed  "s/.*tag: \([^,)]*\).*/\1/")-$$(git rev-parse --short HEAD)" > VERSION
```

Comment:
`git describe` is trying to tell me that this branch is off of `12.1.0-rc.1`. `git log` is behaving better.